src/styles/variables.ts

```diff
@@ -230,10 +233,14 @@ export default {
     mushroomTopHatHeight: 128,
     bankButtonMargin: 23,
     textInputAutoGrowMaxHeight: 115,
+    eReceiptBackgroundImageMinWidth: 217,
 
     searchTypeColumnWidth: 52,
 
+    h36: 36,
     w28: 28,
+    w36: 36,
+    w40: 40,
     w44: 44,
     w80: 80,
     w96: 96,
```

Comment:
NAB, but we should avoid these kinds of variable names in variables.ts, they should technically all be named after their use I believe

Reply:
Couldn't that lead to several variables with the same value? e.g. `dateSearchTableHeaderWidth: 44` and `receiptSearchTableHeaderWidth: 44` I feel like we should generalize some of these values to avoid duplicating variable declarations here